pymagicc/io/__init__.py

```diff
@@ -0,0 +1,613 @@
+import re
+from copy import deepcopy
+from datetime import datetime
+from numbers import Number
+from os.path import basename
+
+import f90nml
+import numpy as np
+from f90nml.namelist import Namelist
+from scmdata import ScmDataFrame
+
+from pymagicc.errors import NoReaderWriterError
+from pymagicc.magicc_time import convert_to_datetime
+from pymagicc.utils import _check_file_exists
+
+from .binout import _BinaryOutReader
+from .compact import _BinaryCompactOutReader, _CompactOutReader
+from .in_mon import (
```

Comment:
```suggestion
from .in_files import (
```

what's `in_mon`

Reply:
This handles .IN/.MON files, but that naming works better